aii-core/src/main/perl/aii-shellfe

```diff
@@ -491,6 +493,10 @@ sub app_options
          HELP    => 'Do not update OS installer (e.g. kickstart) configuration',
          DEFAULT => undef },
 
+       { NAME    => "$PROTECTED_OPTION=s",
+         HELP    => 'confirm id for reinstalling protected nodes',
```

Comment:
it's set in the profile, and printed in the error when you try to use certain options that require it.
we use the buildid, to avoid people to run something from history `ctrl-r` style.
we'll add some documentation after we settle on a name :smile: